### PR TITLE
Veeeery rough first pass at CFR changes

### PR DESCRIPTION
**Pull request by cmc333333**
While we work on getting the data ready, here is a mock-data driven first stab
at displaying changes to the CFR (as described in a proposed or final rule).
No tests yet.

Resolves eregs/notice-and-comment#89 (though without the data).

### CODE REVIEW

**Comment by cmc333333** [comment]
screen shot 2016-04-01 at 5 17 45 pm
screen shot 2016-04-01 at 5 17 36 pm
Did I mention they were rough?

File: regulations/views/preamble.py [hunk 1]
```diff
@@ -84,3 +88,49 @@ def get(self, request, *args, **kwargs):
...
+        if len(label_parts) == 1:
+            context = self.authorities_context(amendments, cfr_part=section)
+        else:
+            context = self.regtext_changes_context(amendments, versions,
+                                                   label_id=section)
+        return TemplateResponse(
+            request=request, template='regulations/cfr_changes.html',
+            context=context)
+
+    def authorities_context(self, amendments, cfr_part):
```
Comment (jmcarp) [inline regulations/views/preamble.py]: Minor, and I'm not even sure I consider this to be a legit code smell, but these helper methods are effectively static.

**Comment by adborden** [comment]
:+1:

**Comment by jmcarp** [comment]
Are we going to use this data to build the table of contents?

Also, I'm thinking about the shortest path from this to functioning comments on CFR changes. Seems like that would involve wrapping this in the minimal markup required to work with the relevant backbone views and adding namespaces for preamble- and CFR-related comments. Course corrections welcome as usual @cmc333333.

**Comment by cmc333333** [comment]
We can either use this data directly, or reconstruct from diffs. The latter approach is used in the existing diff mode's ToC, so I'd lean there.

Re shortest path: agreed on wrapping w/ minimal markup and namespacing. We can also give a "virtual" namespace like `<preamble_id>-cfr-111-12-b` if that'd work better?

**Pull request merged**